openapi/components/schemas/Usage.yaml

```
@@ -0,0 +1,45 @@
+type: object
+required:
+  - subscriptionId
+  - planId
+  - quantity
+properties:
+  subscriptionId:
+    description: ID of the subscription for which a usage is reported.
+    allOf:
+      - $ref: ./ResourceId.yaml
+  planId:
+    description: ID of the plan for which a usage is reported.
+    allOf:
+      - $ref: ./ResourceId.yaml
+  invoiceId:
+    description: ID of the invoice for which usage is considered. This value is added when the invoice with usage is issued.
```

Suggestion to avoid ending the first sentence with preposition 
```suggestion
    description: ID of the invoice to which usage is applied. This value is populated when the invoice is issued.
```